docs/tutorials/custom/custom-base-dog.md

```diff
@@ -24,6 +24,12 @@ cost: 190
 # SMEs: James Otting, Eric Daniels
 ---
 
+<!-- LEARNING GOAL:
+After following this tutorial, you will know about Viam's modular platform and be able to identify if you need to create your own modular resource for your base and create and use that resource.
```

Comment:
Nit: platform means arch to many users, esp as Registry decisions often include "which platform do I download for". Recommend `resources`:

```suggestion
After following this tutorial, you will know about Viam's modular resources and be able to identify if you need to create your own modular resource for your base, as well as how to create and use that resource.
```

Reply:
```suggestion
After following this tutorial, you will know about Viam's modules and be able to identify if you need to create your own modular resource for your base, as well as how to create and use that resource.
```